test/index.js

```diff
@@ -616,8 +616,9 @@ describe('def', function() {
 
     throws(function() { defaultTo(0, 'XXX'); },
            errorEq(TypeError,
-                   '‘defaultTo’ expected a value of type (Nullable Number) ' +
-                   'as its second argument; received "XXX"'));
+                   '‘defaultTo’ expected its first and second arguments to ' +
+                   'be of the same type; the first is of type Number' +
+                   ', however the second, "XXX", is of type String'));
```

Comment:
> but it's not easy to express in words

Agreed! I like what you've done there, I'll play around a bit and see what I can come up with.